app/uk/gov/hmrc/taxenrolmentassignmentfrontend/logging/LoggingEvent.scala

```diff
@@ -32,15 +32,28 @@ trait LoggingEventInfo {
     )
   )
 
-  def logSingleOrMultipleAccountHolderAssignedEnrolment(
+  def logSingleAccountHolderAssignedEnrolment(
     credentialId: String,
     nino: Nino
   ): LoggingEvent =
     Info(
       Event(
         "[AccountCheckController][silentEnrol]",
         details = Some(
-          s"$hmrcPTKey enrolment assigned to credential $credentialId which has a single or multiple accounts with nino ${nino.nino}"
+          s"$hmrcPTKey enrolment assigned to single account credential $credentialId with nino ${nino.nino}"
+        )
+      )
+    )
+
+  def logMultipleAccountHolderAssignedEnrolment(
+    credentialId: String,
+    nino: Nino
+  ): LoggingEvent =
+    Info(
+      Event(
+        "[AccountCheckController][silentEnrol]",
```

Comment:
silentEnrol???
Only single account should be silently enrol. Multiple accoutn should see the interstitial page.

Reply:
It was going to the redirect but this wording in the logging was there before the IV removal and for the SINGLE_OR_MULTIPLE even when it was going to the interstitial, I changed it now to be the wrapping it's called in which I assume is what it was originally designed to do